app/oess_pull_azure_interfaces.pl

```diff
@@ -65,6 +74,8 @@ sub reconcile_oess_endpoints {
     my $db = shift;
     my $endpoints = shift;
     my $azure_connections = shift;
+    my $cloud_interconnect_id = shift;
+    my $logger = Log::Log4perl->get_logger('OESS.Cloud.Azure.Syncer');
```

Comment:
Instead of calling `get_logger` multiple times, just use the global `$logger` variable declared on line 21.

Reply:
now using global throughout